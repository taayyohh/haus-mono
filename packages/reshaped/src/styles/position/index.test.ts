import getPositionStyles from "./index";

describe("Styles/Position", () => {
	test("handles value", () => {
		expect(getPositionStyles("absolute")).toMatchSnapshot();
	});

	test("handles undefined value", () => {
		expect(getPositionStyles()).toMatchSnapshot();
	});

	test("handles responsive value", async () => {
		expect(getPositionStyles({ s: "absolute", l: "fixed" })).toMatchSnapshot();
	});
});
