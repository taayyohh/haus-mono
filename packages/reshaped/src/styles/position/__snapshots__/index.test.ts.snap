// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Styles/Position > handles responsive value 1`] = `
{
  "classNames": [
    "_--position-absolute_aeaee9",
    "_--position-fixed--l_aeaee9",
  ],
}
`;

exports[`Styles/Position > handles undefined value 1`] = `null`;

exports[`Styles/Position > handles value 1`] = `
{
  "classNames": [
    "_--position-absolute_aeaee9",
  ],
}
`;
